Codecademy/gamut
# feat(Icons): Add DiscordOutlineIcon

### Overview



Add DiscordOutlineIcon.



<img width="300" alt="Screenshot 2024-06-04 at 3 40 28 PM" src="https://github.com/Codecademy/gamut/assets/14988121/309ef493-d540-488f-8480-c39e7dfecd83">

### PR Checklist

- [x] Related to designs: https://www.figma.com/design/D7E3nISfSsg84ONPR8atV1/Study-Groups?node-id=177-23894&t=0t6f4bhdlDauWLwN-4
- [x] Related to JIRA ticket: [LEGO-3905](https://codecademy.atlassian.net/browse/LEGO-3905)

#### Testing Instructions



For both portal-app and monolith:
1. Log in with a user who has the Optimizely test user role
2. Check the icon for the Discord link under the Community dropdown in the global header

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/37835)  | [Monolith Env](https://pr-37835-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/6488)  | [Portal Env](https://tengu.codecademy.com/learn?PR_ENV=portal-app-pr-6488)   |




[LEGO-3905]: https://codecademy.atlassian.net/browse/LEGO-3905?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
